docs/monitoring/metrics.md

```diff
@@ -856,66 +918,80 @@ Thus, in order to infer the metric identifier:
 <table class="table table-bordered">
   <thead>
     <tr>
-      <th class="text-left" style="width: 20%">Scope</th>
-      <th class="text-left" style="width: 30%">Metrics</th>
-      <th class="text-left" style="width: 50%">Description</th>
+      <th class="text-left" style="width: 18%">Scope</th>
+      <th class="text-left" style="width: 26%">Metrics</th>
+      <th class="text-left" style="width: 48%">Description</th>
+      <th class="text-left" style="width: 8%">Type</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <th rowspan="7"><strong>Task</strong></th>
       <td>currentLowWatermark</td>
-      <td>The lowest watermark this task has received.</td>
+      <td>The lowest watermark this task has received (in milliseconds).</td>
```

Comment:
AFAIK the timeunit for watermarks depends on the watermark assigned used, so it may not be milliseconds. @aljoscha

Reply:
I see that `currentLowWatermark` is derived from the `Watermark.getTimestamp ()` method,

```
        /**
	 * Returns the timestamp associated with this {@link Watermark} in milliseconds.
	 */
	public long getTimestamp() {
		return timestamp;
	}
```
so  `currentLowWatermark` must be milliseconds ?